src/components/accessibility/screen_reader_status/screen_reader_status.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import { mount, render } from 'enzyme';
+
+import { findTestSubject } from '../../../test';
+
+import { EuiScreenReaderStatus } from './screen_reader_status';
+
+const content = (
+  <p>
+    This paragraph is not visibile to sighted users but will be read by
```

Comment:
Minor optional typo fix
```suggestion
    This paragraph is not visible to sighted users but will be read by
```